service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/EncounterComponentsMapper.java

```diff
@@ -158,4 +161,8 @@ private String mapProcedureRequest(Resource resource) {
     private String mapReferralRequest(Resource resource) {
         return requestStatementMapper.mapReferralRequestToRequestStatement((ReferralRequest) resource, IS_NESTED);
     }
+
+    private String mapDiagnosticReport(Resource resource) {
```

Comment:
Would be good if we could include this mapping in the following test:
`EncounterComponentsMapperTest.When_MappingEncounterComponents_Expect_ResourceMapped()`

I've not checked, but there may be other tests in `EncounterComponentsMapperTest` that we could update too.

Reply:
Have updated `EncounterComponentsMapperTest.When_MappingEncounterComponents_Expect_ResourceMapped()` to include DiagnosticReport. Believe this was the only test to be updated in EncounterComponentMappersTest class.

Reply:
Great - thank you 👍